etc/config/c++.amazon.properties

```diff
@@ -420,6 +420,119 @@ compiler.clang_reflection.semver=(reflection)
 compiler.clang_reflection.options=-std=c++20 -freflection-ts -stdlib=libc++
 compiler.clang_reflection.notification=Experimental Reflection Support; see <a href="https://github.com/cplusplus/reflection-ts" target="_blank" rel="noopener noreferrer">github<sup><small class="fas fa-external-link-alt opens-new-window" title="Opens in a new window"></small></sup></a>
 
+group.clangx86assert.compilers=clang26assert:clang27assert:clang28assert:clang29assert:clang30assert:clang31assert:clang32assert:clang33assert:clang34assert:clang35assert:clang36assert:clang37assert:clang38assert:clang39assert:clang40assert:clang50assert:clang60assert:clang70assert:clang80assert:clang90assert:clang100assert:clang110assert:clang120assert:clang130assert:clang140assert:clang150assert:clang160assert
+group.clangx86assert.intelAsm=-mllvm --x86-asm-syntax=intel
+group.clangx86assert.options=
+group.clangx86assert.groupName=Clang x86-64 (assertions)
+group.clangx86assert.instructionSet=amd64
+group.clangx86assert.baseName=x86-64 clang
+group.clangx86assert.isSemVer=true
+group.clangx86assert.compilerType=clang
+group.clangx86assert.unwiseOptions=-march=native
+group.clangx86assert.supportsPVS-Studio=true
+group.clangx86assert.supportsSonar=true
+group.clangx86assert.licenseName=LLVM Apache 2
+group.clangx86assert.licenseLink=https://github.com/llvm/llvm-project/blob/main/LICENSE.TXT
+group.clangx86assert.licensePreamble=The LLVM Project is under the Apache License v2.0 with LLVM Exceptions
+group.clangx86assert.supportsBinaryObject=true
+group.clangx86assert.compilerCategories=clang
```

Comment:
Most of this could be shared with &clang. So we could have a clang group that declares common props and include only 2 subgroups: the existing clang compilers (current `&clang` ) and the new `&clangx86assert`.

Reply:
Is there an example I can follow?

Reply:
Something like what we have for cross compilers: https://github.com/compiler-explorer/compiler-explorer/blob/d1bb24820662126d082de765e93d0da8a18541ff/etc/config/c.amazon.properties#L879
ppcs have even a deeper nesting : https://github.com/compiler-explorer/compiler-explorer/blob/d1bb24820662126d082de765e93d0da8a18541ff/etc/config/c.amazon.properties#L1176

Reply:
I decided not to touch existing Clang compilers, and also avoid inventing a new name for them (`clangx86release`?). Is that fine by you?

Reply:
100% correct. The compiler names must not change as they are the ids used in shared links. Only the groups can be renamed/moved around.
It's already nice that you took the time to reduce a bit your new properties, thanks very much !